.github/workflows/update-nightlies.yaml

```diff
@@ -33,25 +33,56 @@ jobs:
         - net-contour
         - net-istio
         - net-kourier
+        - eventing-kafka-broker
 
         # Map to nightly-specific parameters.
         include:
         - nightly: net-certmanager
           module: knative.dev/net-certmanager
           directory: ./third_party/cert-manager-latest
           files: net-certmanager.yaml
+          bucket: net-certmanager
+          repository: knative/serving
+          fork: serving
+          channel: net-certmanager
+          assignee: @knative/networking-wg-leads
         - nightly: net-contour
           module: knative.dev/net-contour
           directory: ./third_party/contour-latest
           files: net-contour.yaml contour.yaml
+          bucket: net-contour
+          repository: knative/serving
+          fork: serving
+          channel: net-contour
+          assignee: @knative/networking-wg-leads
         - nightly: net-istio
           module: knative.dev/net-istio
           directory: ./third_party/istio-latest
           files: net-istio.yaml
+          bucket: net-istio
+          repository: knative/serving
+          fork: serving
+          channel: net-istio
+          assignee: @knative/networking-wg-leads
```

Comment:
This is a lot of new knobs 🤔

Reply:
+1, what is all this?

Reply:
- serving pulls dependencies from downstream we need to do the other way around, so nightly name and bucket don't match.
- we don't have a dedicated slack channel, so nightly and channel don't match too.
- the assignee is different.

I can remove the fork and derive it from the repository but I wanted to reduce bash logic and be more explicit.